editors/README.md

```diff
@@ -0,0 +1,11 @@
+# ⚓ Oxc Editor Plugins
+
+## Official
+
+[VSCode](https://github.com/oxc-project/oxc/tree/main/editors/vscode)
+
+[(Neo)Vim](https://github.com/oxc-project/coc-oxc)
```

Comment:
https://github.com/oxc-project/oxc-intellij-plugin

Reply:
I don't use IntelliJ, Is that plugin in a working state? These are only working extensions.

Reply:
That repos seems abandoned 😅